static/js/lib/util.js

```diff
@@ -148,3 +148,11 @@ export const flatZip = R.compose(
   R.flatten,
   R.zip
 )
+
+export const capitalize = R.converge(R.concat(), [
+  R.compose(
+    R.toUpper,
+    R.head
+  ),
+  R.tail
+])
```

Comment:
what does this function do differently than `_.capitalize`? I tried it out in a node repl and `_.capitalize` seems to return the same thing

Reply:
In an earlier PR review, Nathan said I should avoid using lodash for new functionality and use ramda instead, so this is the ramda equivalent.

Reply:
ah ok, I was just wondering because I saw some other uses of `_.capitalize` around as well.